replica/diff_disk.go

```diff
@@ -311,7 +312,7 @@ func (d *diffDisk) lookup(sector int64) (byte, error) {
 	}
 
 	target := d.location[sector]
-	if target == 0 {
+	if target == 0 || d.preloadStatus == "Started" || d.preloadStatus == "Error" {
```

Comment:
better would be d.prelodStatus != "Done"

Reply:
ok